# Copyright 2022 Ellis Ratner (eratner@berkeley.edu)

# Name of the obstacle.
string name

# Type of the obstacle (currently only "rectangle" supported)
string type

# Pose of the obstacle.
geometry_msgs/Pose pose

# Size of the obstacle.
geometry_msgs/Point size

# If set to true, obstacle can move.
bool moveable

# The following parameters only apply to moveable obstacles.
float64 mass
float64 restitution_coeff
float64 friction_coeff

---
bool success